tests/cases/standalone/common/copy/copy_from_fs_parquet.sql

```diff
@@ -40,6 +40,14 @@ Copy with_limit_rows FROM '/tmp/demo/export/parquet_files/' WITH (MAX_INSERT_ROW
 
 select count(*) from with_limit_rows;
 
+CREATE TABLE with_limit_rows_segment(host string, cpu double, memory double, ts timestamp time index);
+
+Copy with_limit_rows_segment FROM '/tmp/demo/export/parquet_files/' LIMIT 2;
+
+select count(*) from with_limit_rows_segment;
+
+Copy with_limit_rows_segment FROM '/tmp/demo/export/parquet_files/' LIMIT -1;
```

Comment:
We should test providing an invalid option to the with clause. But I think we don't need to support this option once the limit syntax is implemented.
```
Copy with_limit_rows FROM '/tmp/demo/export/parquet_files/' WITH (MAX_INSERT_ROWS = 2);
```